templates/article.json

```diff
@@ -19,8 +19,8 @@
      "block_order": [
        "featured_image",
        "title",
-       "content",
-       "share"
+       "share",
+       "content"
     ]
    }
  },
```

Comment:
Is there a console log I am missing? 🤔  I dont see it on `main`

https://screenshot.click/10-38-3enf1-lhc0l.png

Reply:
oh this is coming from the blog post itself due to how it's setup in the admin: ![](https://screenshot.click/21-08-dml7z-pe5kk.png)